### DISCUSSION TITLE
Shortcut for switching between tabs in Sidebar Panels

### DISCUSSION THREAD

**Discussion by senntore** [zen-browser/desktop#4022] [Ideas]
After opening the sidebar using my allotted shortcut, I have to use mouse or touchpad to switch between it's tabs.
I didn't find shortcuts related to sidebar panels(aside from it's toggle state) in the settings. It would be great if we can set shortcuts for them.

**Comment by tzinm** [comment]
The sidepanel doesn't seem to have any shortcuts.

Some of the useful shortcuts:

* Move between 'pinned' tabs
* Crate new 'pinned' tabs
* Delete a 'pinned' tab
* Movee between the sidepanel and the 'main' window
